### DISCUSSION TITLE
My name is asadbek

### DISCUSSION THREAD

**Discussion by Doimjonov-Asadbek** [community/community#38555] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm from Uzbekistan

**Comment by Doimjonov-Asadbek** [comment, marked as answer]
Hello

**Comment by subidit** [comment]
Dude you wont get Galaxy brain if you mark your own reply as answers....
~mark my replys as answers... then i'll ask 2 more where you could reply.~

Ask two new questions and I'll do the same...
